l2tdevtools/build_helpers/dpkg.py

```diff
@@ -274,11 +286,17 @@ def _RemoveOlderOriginalSourcePackage(self, project_name, project_version):
     """
     filenames_to_ignore = '^{0:s}_{1!s}.orig.tar.gz'.format(
         project_name, project_version)
+    if version_suffix and distribution:
```

Comment:
I opt to make this an if-else instead of an set-if-override

Reply:
Done